### PR TITLE
Add Chef

### PR DESCRIPTION
**Pull request by afiune**
Chef lets you develop, deploy and manage infrastructure, run-time environments and applications.
https://www.chef.io/

Signed-off-by: Salim Afiune <afiune@chef.io>

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -93,6 +93,11 @@
   logo: coturnix.png
   how: "Machine learning and optimization for minimizing energy consumption in buildings."
 -
+  name: delivery-cli
```
Comment (smith) [inline _data/users.yml]: This is usually the name of the product/company, so it should probably be "Chef"

**Comment by afiune** [comment]
Putting this on hold for now..

File: _data/users.yml (hunk 1 above)
Comment (afiune) [inline _data/users.yml]: Nice! Thanks Nathan Ill fix it..

**Comment by aturon** [comment]
Yay, thank you!

**Pull request merged**